ts/api/provider_frame_elem.ts

```diff
@@ -29,19 +31,38 @@ const DEFAULT_FRAME_CSS = `
   position: fixed;
   border: none;
   z-index: 9999;
+}
+
+.openyolo-visible.bottomSheet {
   bottom: 0;
+  width: 100%;
+  height: 320px;
+}
+
+.openyolo-visible.navPopout {
+  top: 0;
+  width: 100%;
+  height: 320px;
+}
+
+.openyolo-visible.fullScreen {
+  top: 0;
   left: 0;
   width: 100%;
+  height: 100%;
 }
 
-@media (min-width:801px) {
-  .openyolo-visible {
-    position: fixed;
-    left: auto;
+@media only screen and (min-device-width:321px) {
+  .openyolo-visible.bottomSheet {
+    left: 50%;
+    width: 320px;
+    margin-left: -160px;
+  }
+
+  .openyolo-visible.navPopout {
     top: 16px;
```

Comment:
It may be better to leave the IFrame anchored at the top right corner, without margin, so that each provider could implement entrance animation (like sliding from the top) and set their own margin. What do you think?